automation/travisci-test.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -e
```

Comment:
-ex maybe would give almost all info we want ?

Reply:
This PR does one thing as described in the commit message.
If you this it is needed, open a new PR and reason for it. I personally do not think it is needed.